resources/istio-resources/README.md

```diff
@@ -0,0 +1,10 @@
+# Istio Resources
+
+## Overview
+
+[Istio](https://istio.io/) is an open-source service mesh providing a uniform way to integrate microservices, manage traffic flow across microservices, enforce policies, and aggregate telemetry data.
+
+The Istio Resources Helm chart consists of:
+
+- Mutual TLS (mTLS) configuration enabling mTLS cluster-wide in a STRICT mode
+- Istio monitoring configuration details
```

Comment:
Should we mention that the dashboards reflect the official istio grafana dashboards?: https://istio.io/latest/docs/ops/integrations/grafana/

Reply:
Agreed with @strekm will be done in follow-up PR